src/main/scala/nl/hugo/redbook/ch7/Par.scala

```diff
@@ -0,0 +1,114 @@
+package nl.hugo.redbook.ch7
+
+import java.util.concurrent._
+import language.implicitConversions
+
+object Par {
+  type Par[A] = ExecutorService => Future[A]
+
+  def run[A](s: ExecutorService)(a: Par[A]): Future[A] = a(s)
```

Comment:
Copied from original exercise. 

https://github.com/fpinscala/fpinscala/blob/master/exercises/src/main/scala/fpinscala/parallelism/Par.scala#L9

The book introduces the Non-Blocking interface at page 115, but then switches back to the blocking interface in the grey section at page 120. It is indeed confusing.